src/renderer/src/components/song/SongImage.tsx

```diff
@@ -1,41 +1,35 @@
-import { Accessor, Component, createEffect, createSignal, onCleanup, onMount } from 'solid-js';
+import { Component, createEffect, createSignal, JSX, onCleanup, onMount } from "solid-js";
 import defaultBackground from "../../assets/osu-default-background.jpg";
-import { availableResource, getResourcePath } from '../../lib/tungsten/resource';
-import "../../assets/css/song/song-image.css";
-
-
+import { availableResource, getResourcePath } from "../../lib/tungsten/resource";
 
 const SET_SOURCE_EVENT = "set-source";
 const GLOBAL_GROUP = "global-group";
 
 const observers = new Map<string, IntersectionObserver>();
 
 type SongImageProps = {
-  src: string | undefined | Accessor<string | undefined>,
-  group?: string,
-  instantLoad?: boolean,
-}
-
+  src: string | undefined;
+  group?: string;
+  instantLoad?: boolean;
+} & JSX.IntrinsicElements["div"];
 
-
-const SongImage: Component<SongImageProps> = props => {
-  const [bg, setBg] = createSignal<string | undefined>();
+const SongImage: Component<SongImageProps> = (props) => {
   const [src, setSrc] = createSignal(defaultBackground);
   let image;
 
-  const setSource = evt => {
+  const setSource = (evt) => {
     setSrc(evt.detail);
     delete image.dataset.eventHandler;
   };
 
-
-
   createEffect(async () => {
-    const b = bg();
+    const b = props.src;
```

Comment:
Will it still execute every time the background changes? If I remember correctly I used `const b = bg()` to state that this effect shall run every time signal bg changes